### PR TITLE
chore: remove CI files

### PR DESCRIPTION
**Pull request by vmx**
The only CI we are using is Jenkins, remove files from all other services.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,6 @@
 [![](https://img.shields.io/badge/project-IPLD-blue.svg?style=flat-square)](http://github.com/ipld/ipld)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 [![Coverage](https://coveralls.io/repos/github/ipld/js-ipld-git/badge.svg?branch=master)](https://coveralls.io/github/ipld/js-ipld-git?branch=master)
-[![Travis](https://travis-ci.org/ipld/js-ipld-git.svg?branch=master)](https://travis-ci.org/ipld/js-ipld-git)
-[![Circle](https://circleci.com/gh/ipld/js-ipld-git.svg?style=svg)](https://circleci.com/gh/ipld/js-ipld-git)
```
Comment (magik6k) [inline README.md:8]: There is a jenkins badge (see go-ipfs for examples)

**Review by magik6k — COMMENTED** [review]
1 nit, then LGTM

File: README.md (hunk 1 above)
Comment (vmx) [reply README.md:8]: Oh nice, I didn't know that. I'll add that then.

**Review by magik6k — APPROVED** [review]

**Comment by vmx** [comment]
CI passed, it's only left-overs from old CI.

**Pull request merged**